enterprise/models.py

```diff
@@ -395,6 +401,26 @@ def serialized(self):
         from enterprise.api.v1 import serializers  # pylint: disable=import-outside-toplevel
         return serializers.EnterpriseCustomerSerializer(self).data
 
+    @property
+    def branding_configuration(self):
+        """
+        Return the customer branding configuration or a default object if none exists.
+        """
+        try:
+            return self._branding_configuration
+        except EnterpriseCustomerBrandingConfiguration.DoesNotExist:
+            default_branding = EnterpriseCustomerBrandingConfiguration(
+                enterprise_customer=self,
+                primary_color='#1a337b',
+                secondary_color='#d7e3fc',
+                tertiary_color='#007d88',
```

Comment:
Nit: maybe store these in variables and leave a comment about where you got them from?